README.md

```diff
@@ -55,7 +56,7 @@ To view a specific page example, choose one of the following options and enter t
 ```javascript
 window.postMessage({
```

Comment:
The three JSONs in the README are not aligned properly. Let's fix the indentation to make them easy to read.